compiler/sempass2.nim

```diff
@@ -9,7 +9,7 @@
 
 import
   intsets, ast, astalgo, msgs, renderer, magicsys, types, idents, trees,
-  wordrecg, strutils, options, guards, writetracking
+  wordrecg, strutils, options, guards, writetracking, lookups
```

Comment:
I think ``isInfixAs`` should not be in ``lookups``.

Reply:
moved to ast.nim